### ISSUE TITLE
Whitelist not working

### ISSUE THREAD

**Issue by jafri** [EOSIO/eosio-reference-ios-authenticator-app#18]
https://github.com/EOSIO/eosio-reference-ios-authenticator-app/blob/7a64ca06609f627125b9b3b9ed1630d77e09f981/Eosio/Manifest%20%26%20Metadata/AppManifest.swift#L37

I believe an empty whitelist comes in as "", not "0". 

Here is chrome authenticator code for reference:
https://github.com/EOSIO/eosio-reference-chrome-extension-authenticator-app/blob/a6d77e0678aa1479f3b0116b81ed82d14eec32f1/src/utils/manifest/ManifestValidator.ts#L126

Can someone confirm this?

**Comment by TaraTritt** [comment]
Thanks @jafri. Yes as per the Manifest Specification, [wildcards are denoted by empty string ("") in the whitelist field](https://github.com/EOSIO/manifest-spec#manifest-fields).

**Comment by brandonfancher** [comment]
@jafri Good catch. Looks like this was a regression in the Reference iOS Authenticator App.
